Common/Brokerages/TDAmeritradeBrokerageModel.cs

```diff
@@ -29,7 +29,7 @@ public class TDAmeritradeBrokerageModel : DefaultBrokerageModel
         /// <summary>
         /// Array's TD Ameritrade supports security types 
         /// </summary>
-        private readonly HashSet<SecurityType> _supportSecurityTypes = new (new [] { SecurityType.Equity, SecurityType.Option, SecurityType.Future });
+        private readonly HashSet<SecurityType> _supportSecurityTypes = new (new [] { SecurityType.Equity, SecurityType.Option });
```

Comment:
I believe it supports `SecurityType.IndexOption` (see [TDAmeritradeSymbolMapper.cs#L43](https://github.com/QuantConnect/Lean.Brokerages.TDAmeritrade/blob/master/QuantConnect.TDAmeritrade/TDAmeritradeSymbolMapper.cs#L43)).

Reply:
Added `Index `and `IndexOption `type.